.setup/bin/sample_courses/models/component.py

```diff
@@ -56,9 +54,10 @@ def __init__(self, component, order) -> None:
     def create(self, g_id, conn, table, mark_table) -> None:
         ins = table.insert().values(g_id=g_id, gc_title=self.title, gc_ta_comment=self.ta_comment,
                                     gc_student_comment=self.student_comment,
-                                    gc_lower_clamp=self.lower_clamp, gc_default=self.default, gc_max_value=self.max_value,
-                                    gc_upper_clamp=self.upper_clamp, gc_is_text=self.is_text,
-                                    gc_is_peer=self.is_peer_component, gc_order=self.order, gc_page=self.page)
+                                    gc_lower_clamp=self.lower_clamp, gc_default=self.default,
+                                    gc_max_value=self.max_value, gc_upper_clamp=self.upper_clamp,
+                                    gc_is_text=self.is_text, gc_is_peer=self.is_peer_component,
+                                    gc_order=self.order, gc_page=self.page)
```

Comment:
Same comment here.  This diff is pretty hard to read, and future diffs will be similarly difficult to read.  You could consider putting each argument on a separate line.